src/frameworks/next/index.ts

```diff
@@ -16,6 +16,7 @@ import { parser } from "stream-json";
 import { pick } from "stream-json/filters/Pick";
 import { streamObject } from "stream-json/streamers/StreamObject";
 import { fileExistsSync } from "../../fsutils";
+import * as esbuild from "esbuild";
```

Comment:
The concern here is that we'd need to pin esbluild, this led to numerous problems before since firebase-tools would be pinned to versions of esbuild that didn't support the node version developers were on or even their architecture (apple silicon was the source of many issues filed). If we could do something clever like how NPX installs on demand, then dynamically load from that path, then we'd have the best of both worlds IMO